vault/resource_namespace.go

```diff
@@ -48,31 +50,80 @@ func namespaceResource() *schema.Resource {
 				Optional:    true,
 				Description: "The fully qualified namespace path.",
 			},
+			consts.FieldCustomMetadata: {
+				Type:     schema.TypeMap,
+				Computed: true,
+				Optional: true,
+				Description: "A map of arbitrary string to string valued user-provided " +
```

Comment:
nit: suggest a more succinct description.
```suggestion
				Description: "Custom metadata describing this namespace. Value type is map[string]string." +
```